src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -1388,6 +1366,7 @@ def sendPilotSubmissionAccounting(self, siteName, ceName, queueName, numTotal, n
         if not result["OK"]:
             self.log.error("Error in Commit:" + result["Message"])
             return result
+        return S_OK()
```

Comment:
Well spotted!

Looks like errors coming from `sendPilotSubmissionAccounting` and `sendPilotSubmissionMonitoring` would not be handled properly. For instance here: https://github.com/DIRACGrid/DIRAC/blob/ea90dc3e4c14e3e872be932ed97e41f169aff27e/src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py#L741